controllers/noobaa-rules.yaml

```diff
@@ -59,16 +61,14 @@
            system_type: NooBaa
            system_vendor: Red Hat
```

Comment:
these 2 labels don't seem useful and aren't consistent with the rest of the recording rules.